docs/configuration/appointment/jcc.rst

```diff
@@ -0,0 +1,15 @@
+===
+JCC
+===
+
+1. In Open Forms navigate to: **Configuration** > **JCC configuration**
+2. Click the **Green Plus Button** and fill in the following details:
+
+   * **Label**: JCC
+   * **Ontvangende organisatie**: Open Forms
+   * **Ontvangende applicatie**: Open Forms
+   * **Versturende organisatie**: Horstaandemaas
+   * **Versturende applicatie**: Afspraak Acceptatie
```

Comment:
Should these be updated to something like *Your Ontvangende organisatie*,  *Your Ontvangende applicate*?

Reply:
Well, they should not contain any REAL reference AND these are not actually needed for JCC.

So, please change all values to: `*Unused*`

Reply:
Okay.  Should we make a note about needing to fill _something_ in?  These fields are required on the model so the user will need to put something in these fields.

We could also not make these fields required though I don't know if we want to make that model change?